common/add_operation_error.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+parent: Common
+---
+
+# Add Operation Error
+
+Understanding the `AddOperationError` exception.
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .note }
+> _An **Add** operation targeting an existing profile could end up effectively performing an **Alter** operation on an existing profile. Pyracf will always raise an `AddOperationError` and **refuse** to perform the requested operation to bring attention to this condition._
+
+&nbsp;
+
+Prior to executing an **Add** operation, a **Profile Extract** is attempted to evaluate whether the profile already exists. If the **Return Code** and the **Messages** returned by the **Profile Extract** operation indicate that the profile already exists, an `AddOperationError` will be raised and the requested **Add** operation will **NOT** be executed. An `AddOperationError` can be handled as follows.
+
+###### Python Script
+```python
+from pyracf import UserAdmin
+from pyracf import AddOperationError
+
+user_admin = UserAdmin()
+
+try:
+    user_admin.add("squidwrd")
+except AddOperationError as e:
+    print(e.message)
+```
+
+###### Console Output
+```console
+Security request made to IRRSMO00 failed.
```

Comment:
Sample console output needs to be updated according to recent updates to this exception.

Reply:
agree, change now made.